app/src/main/java/com/example/sharingang/items/ItemRepositoryModule.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.sharingang.items
+
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ViewModelComponent
+
+@Module
+@InstallIn(ViewModelComponent::class)
+/**
+ * Tells Hilt to use an instance of FirestoreItemRepository when we require an ItemRepository
+ */
```

Comment:
Nitpick: I think the convention is to put doc comments *before* annotations, and we already did it that way in `ItemsViewModel`